File: claude/nivel basico/ruby/suma.rb
def suma_numeros(a, b)
    return a + b
  end
  
  # Solicitar valores al usuario
  print "Ingrese el primer número: "
  num1 = gets.chomp.to_f
  
  print "Ingrese el segundo número: "
  num2 = gets.chomp.to_f
  
  # Calcular y mostrar el resultado
  resultado = suma_numeros(num1, num2)
  puts "La suma de #{num1} y #{num2} es: #{resultado}"

File: claude/nivel basico/javascript/suma.js
/**
 * Función que calcula la suma de dos números.
 * 
 * @param {number} a - Primer número
 * @param {number} b - Segundo número
 * @returns {number} Resultado de la suma
 */
function sumaNumeros(a, b) {
    return a + b;
}

// Solicitar valores al usuario
const num1 = parseFloat(prompt("Ingrese el primer número:"));
const num2 = parseFloat(prompt("Ingrese el segundo número:"));

// Calcular y mostrar el resultado
const resultado = sumaNumeros(num1, num2);
console.log(`La suma de ${num1} y ${num2} es: ${resultado}`);